src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -48,6 +48,48 @@
  * &lt;module name=&quot;DefaultComesLast&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
```

Comment:
make it single-line, for other example as well

Reply:
done.